client/alias_options.go

```diff
@@ -0,0 +1,130 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import "github.com/milvus-io/milvus-proto/go-api/v2/milvuspb"
+
+// CreateCollectionOption is the interface builds CreateAliasRequest.
+type CreateAliasOption interface {
+	Request() *milvuspb.CreateAliasRequest
+}
+
+type createAliasOption struct {
+	collectionName string
+	aliasName      string
+}
+
+func (opt *createAliasOption) Request() *milvuspb.CreateAliasRequest {
+	return &milvuspb.CreateAliasRequest{
+		CollectionName: opt.collectionName,
+		Alias:          opt.aliasName,
+	}
+}
+
+func NewCreateAliasOption(collectionName, aliasName string) *createAliasOption {
+	return &createAliasOption{
+		collectionName: collectionName,
+		aliasName:      aliasName,
+	}
+}
+
+// DescribeAliasOption is the interface builds DescribeAliasOption.
+type DescribeAliasOption interface {
+	Request() *milvuspb.DescribeAliasRequest
+}
+
+type describeAliasOption struct {
+	aliasName string
+}
+
+func (opt *describeAliasOption) Request() *milvuspb.DescribeAliasRequest {
+	return &milvuspb.DescribeAliasRequest{
+		Alias: opt.aliasName,
+	}
+}
+
+func NewDescribeAliasOption(alias string) *describeAliasOption {
+	return &describeAliasOption{
+		aliasName: alias,
+	}
+}
+
+// DropAliasOption is the interface builds DropAliasRequest.
+type DropAliasOption interface {
+	Request() *milvuspb.DropAliasRequest
+}
+
+type dropAliasOption struct {
+	aliasName string
+}
+
+func (opt *dropAliasOption) Request() *milvuspb.DropAliasRequest {
+	return &milvuspb.DropAliasRequest{
+		Alias: opt.aliasName,
+	}
+}
+
+func NewDropAliasOption(alias string) *dropAliasOption {
+	return &dropAliasOption{
+		aliasName: alias,
+	}
+}
+
+// AlterAliasOption is the interface builds AlterAliasRequest.
+type AlterAliasOption interface {
+	Request() *milvuspb.AlterAliasRequest
+}
+
+type alterAliasOption struct {
+	aliasName      string
+	collectionName string
+}
+
+func (opt *alterAliasOption) Request() *milvuspb.AlterAliasRequest {
+	return &milvuspb.AlterAliasRequest{
+		Alias:          opt.aliasName,
+		CollectionName: opt.collectionName,
+	}
+}
+
+func NewAlterAliasOption(alias, collection string) *alterAliasOption {
+	return &alterAliasOption{
+		aliasName:      alias,
+		collectionName: collection,
+	}
+}
+
+// ListAliasesOption is the interface builds ListAliasesRequest.
+type ListAliasesOption interface {
+	Request() *milvuspb.ListAliasesRequest
+}
+
+type listAliasesOption struct {
+	collectionName string
+}
+
+func (opt *listAliasesOption) Request() *milvuspb.ListAliasesRequest {
+	return &milvuspb.ListAliasesRequest{
+		CollectionName: opt.collectionName,
+	}
+}
+
+func NewListAliasesOption(collection string) *listAliasesOption {
```

Comment:
Maybe it would be better to change the parameter `collection` to `collectionName`. Keep it the same as func NewCreateAliasOption, and keep it the same as pymilvus

Reply:
fixed